esx_service/cli/vmdkops_admin.py

```diff
@@ -914,7 +914,9 @@ def generate_tenant_ls_rows(tenant_list):
         uuid = tenant.id
         name = tenant.name
         description = tenant.description
-        if not tenant.default_datastore_url or tenant.default_datastore_url == auth_data_const.DEFAULT_DS_URL:
+        # "default_datastore_url" should always be set, and cannot be empty
+        # it can only happen when DB has some corruption
```

Comment:
I think we should log an error here and also set the default_datastore to empty since we need to show something in this field.